staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt-config.go

```diff
@@ -0,0 +1,284 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filters
+
+import (
+	"math"
+	"sort"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+	cache "k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	rmtypesv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+)
+
+// initialSync does the initial setup of the configuration of the
+// reqmgmt filter.  It either succeeds and returns `true` or logs an
+// error and returns `false.
+func (reqMgmt *requestManagement) initialSync() bool {
+	rms := &RMState{
+		flowSchemas:         make(FlowSchemaSeq, 0),
+		priorityLevelStates: make(map[string]*PriorityLevelState),
+	}
+	reqMgmt.curState.Store(rms)
+	reqMgmt.plInformer.AddEventHandler(reqMgmt)
+	reqMgmt.fsInformer.AddEventHandler(reqMgmt)
+	stopCh := make(chan struct{})
+	go reqMgmt.plInformer.Run(stopCh)
+	go reqMgmt.fsInformer.Run(stopCh)
+	if ok := cache.WaitForCacheSync(stopCh, reqMgmt.plInformer.HasSynced, reqMgmt.fsInformer.HasSynced); !ok {
+		return false
+	}
+	ok := reqMgmt.syncOne()
+	if !ok {
+		klog.Error("Unable to do initial config sync")
```

Comment:
as a convention of golang, https://github.com/golang/go/wiki/Errors#errors, all the error messages should not start w/ a capital letter

Reply:
There is a difference between (a) the string in an `error` and (b) a message written with glog or klog.  Yes, the string in an `error` should start lower case, and that is what the cited doc says.  What we have here is (b), where I think a normal English sentence is appropriate.